cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/TranslationConfiguration.kt

```diff
@@ -420,6 +435,16 @@ private constructor(
             return this
         }
 
+        fun registerFunctionSummaries(functionSummaries: List<File>): Builder {
```

Comment:
Wouldn't it make more sense to make this a File varargs? Then we can either have one file or a list of files instead of having two functions.